src/whylogs/core/model_profile.py

```diff
@@ -48,14 +51,14 @@ def compute_metrics(self, targets,
         Parameters
```

Comment:
pydoc-markdown package we use for extracting dco comments is expecting Sphinx style documentation, but this method is using Numpy style.
I don't care which style we use but we ought to standardize on one, and the documentation for this method is not recognized and gets formatted badly.

Reply:
Don't worry about the doc change, I'll go through and standardize our comment style in one PR